components/config/data.ts

```diff
@@ -0,0 +1,27 @@
+import client from '@/lib/client'
+
+const key = 'external_labeler_dids'
```

Comment:
Distinguishing this as a constant vs. a local variable might be friendly!
```suggestion
const KEY = 'external_labeler_dids'
```

Reply:
I often forget my own convention but normally, I try to only uppercase constants if they are being exported but no strong feelings about it.